plugin/wp-cli-login-server.php

```diff
@@ -287,25 +302,29 @@ private function abort(Exception $e)
      */
     private function magicKey()
     {
-        return self::OPTION . '/' . $this->publicKey;
+        // We need to hash the salt to produce a key that won't exceed the maximum of 64 bytes.
+        $key = sodium_crypto_generichash(wp_salt('auth'));
+        $bin_hash = sodium_crypto_generichash($this->publicKey, $key);
```

Comment:
Again, the terminology is essential, publicKey indicates asymmetric cryptography which is not used at all here. 

It's just the URL endpoint

Reply:
I'd name this something like:

"endpoint" => "route_prefix"

"publicKey" => "provided_magic_link"